.github/workflows/code-quality.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           args: >
             -Dsonar.organization=nhsd-exeter
             -Dsonar.projectKey=uec-dos-api-sfsa
-            -Dsonar.java.binaries=target/classes
+            -Dsonar.java.binaries=target/classes -Dsonar.java.libraries=target/lib/*.jar
```

Comment:
can we pop this onto a new line please just to keep it readable

Reply:
done